aws/table_aws_ec2_application_load_balancer.go

```diff
@@ -172,6 +172,11 @@ func listEc2ApplicationLoadBalancers(ctx context.Context, d *plugin.QueryData, _
 func getEc2ApplicationLoadBalancer(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
 	loadBalancerArn := d.KeyColumnQuals["arn"].GetStringValue()
 
+	// check if arn is empty
+	if loadBalancerArn == "" {
```

Comment:
Do we need this check if we are adding `ValidationError` in `ShouldIgnoreError`?

Reply:
yes, always empty check in get call. API should not be called with empty check.

Reply:
Okay, Thanks!